api/audit/constants.py

```diff
@@ -12,6 +12,7 @@
 FEATURE_STATE_SCHEDULED_MESSAGE = (
     "Flag state / Remote Config value update scheduled for %s for feature: %s"
 )
+FEATURE_STATE_WILL_BE_UPDATED_MESSAGE = "Flag state for feature '%s' will be updated at %s UTC unless the Change Request '%s' is rescheduled or deleted."
```

Comment:
1. IMO, the "unless rescheduled or deleted" clarification is redundant here. The user should see the full picture by looking at all the available audit logs. If we want a more helpful message, consider adding heuristics to determine whether the change was initially scheduled by a new change request, or rescheduled for an existing one.   

2. I'm not a fan of future tense in log messages. Consider improving the wording to something among the lines of

```suggestion
FEATURE_STATE_WILL_BE_UPDATED_MESSAGE = "Flag state for feature '%s' scheduled to update at %s UTC."
```

Reply:
Addressed with 0dd1b0d5984a967dc677623cfba83a62eb9aedb5.